pkg/operator/ceph/file/mds/mds_liveness_probe.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+ROOK_CEPH_MON_HOST="{{ .RookCephMonHost }}"
+ROOK_CEPH_MON_INITIAL_MEMBERS="{{ .RookCephMonInitialMembers }}"
```

Comment:
These should be set in the environment in the container and not via Golang templating.

Reply:
Here I am just echo-ing what envs are set in the mds containers running in my install right now
```
#$> kr exec -it  rook-ceph-mds-myfs-a-54b74f5d75-dwxbm -- bash            ⎈ minikube
Defaulted container "mds" out of: mds, chown-container-data-dir (init)
[root@rook-ceph-mds-myfs-a-54b74f5d75-dwxbm ceph]# echo $ROOK_CEPH_MON_HOST
[v2:10.106.96.242:3300,v1:10.106.96.242:6789]
[root@rook-ceph-mds-myfs-a-54b74f5d75-dwxbm ceph]# echo $ROOK_CEPH_MON_INITIAL_MEMBERS
a
```

Reply:
Okay thanks